connectors/opa/lib/catalog_reader.go

```diff
@@ -57,8 +57,8 @@ func (r *CatalogReader) GetDatasetsMetadataFromCatalog(in *pb.ApplicationContext
 	return datasetsMetadata, nil
 }
 
-func (r *CatalogReader) GetDatasetMetadata(ctx *context.Context, client pb.DataCatalogServiceClient, datasetID string, appID string) (map[string]interface{}, error) {
-	objToSend := &pb.CatalogDatasetRequest{AppId: appID, DatasetId: datasetID}
+func (r *CatalogReader) GetDatasetMetadata(ctx *context.Context, client pb.DataCatalogServiceClient, datasetID string, creds string) (map[string]interface{}, error) {
+	objToSend := &pb.CatalogDatasetRequest{CredentialPath: creds, DatasetId: datasetID}
```

Comment:
just a general query: in the egeria scenario, creds will be empty right ?

Reply:
If no secret is attached to m4dapp, an empty string will be passed to egeria-connector. Otherwise, it will be a pointer to this secret. In any case, egeria-connector ignores this value.

Reply:
Ok thanks